Pilot/python2.6_tests/simple_ssl_server.py

```diff
@@ -0,0 +1,116 @@
+'''This is simple server, which is based on sockets and ssl '''
```

Comment:
Add content of 4 certs as strings.
First step of the intergration test should be creation of those four files 
Last step should be cleanup